.azure-pipelines/pipelines.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   - job: format
     dependsOn: [] # this removes the implicit dependency on previous stage and causes this to run in parallel.
     pool:
-      vmImage: "ubuntu-16.04"
+      vmImage: "ubuntu-18.04"
```

Comment:
QQ: why not 20.04?

Reply:
not provided yet: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops